en_US/advanced/http-api.md

```diff
@@ -1784,3 +1784,32 @@ $ curl -XDELETE --basic -u admin:public 'http://localhost:8081/api/v4/resources/
 
 {"code":0}
 ```
+
+### License
+
+Facilities for managing the EMQX license.
+
+#### POST /api/v4/license/upload
+
+Uploads a new license file to the cluster.  The license is validated, then it is copied to all nodes in the cluster and reloaded.  The new contents are written to the same file path configured in the node, and the old license contents are backed up to a file in the base directory, suffixed by the timestamp when the change occurred.
```

Comment:
what's base directory ?

should be data directory instead ?

Reply:
I've updated to reflect that it's the same base directory as the already configured license path.